### PR TITLE
Add Clova Embeddings for LangChain Community

### PR DESCRIPTION
**Pull request by acho98**
- [ ] **PR title**: "Add Naver ClovaX embedding to LangChain community"
- HyperClovaX is a large language model developed by [Naver](https://clova-x.naver.com/welcome).
It's a powerful and purpose-trained LLM.

- You can visit the embedding service provided by [ClovaX](https://www.ncloud.com/product/aiService/clovaStudio)

- You may get CLOVA_EMB_API_KEY, CLOVA_EMB_APIGW_API_KEY, CLOVA_EMB_APP_ID From
https://www.ncloud.com/product/aiService/clovaStudio

### CODE REVIEW

File: libs/community/langchain_community/embeddings/clova.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                values, "clova_emb_apigw_api_key", "CLOVA_EMB_APIGW_API_KEY"
+            )
+        )
+        values["app_id"] = convert_to_secret_str(
+            get_from_dict_or_env(values, "app_id", "CLOVA_EMB_APP_ID")
+        )
+        return values
+
+    def embed(self, texts: List[str]) -> List[List[float]]:
+        return self.embed_documents(texts)
```
Comment (baskaryan) [inline libs/community/langchain_community/embeddings/clova.py:74]: ooc what's this method for?

Comment (acho98) [reply libs/community/langchain_community/embeddings/clova.py:74]: The embed method was meant to provide a simple interface for embedding multiple texts by calling embed_documents. However, it's functionally redundant. 
I'll remove embed to simplify the code. 

Thank you for the review!

Comment (acho98) [reply libs/community/langchain_community/embeddings/clova.py]: I've removed the redundant embed method as discussed. The changes have been pushed to the branch.

Thank you for your review, @baskaryan!

File: libs/community/langchain_community/embeddings/clova.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    """
+
+    endpoint_url: str = (
+        "https://clovastudio.apigw.ntruss.com/testapp/v1/api-tools/embedding"
+    )
+    """Endpoint URL to use."""
+    model: str = "clir-emb-dolphin"
+    """Embedding model name to use."""
+    clova_emb_api_key: Optional[SecretStr] = None
+    clova_emb_apigw_api_key: Optional[SecretStr] = None
```
Comment (baskaryan) [inline libs/community/langchain_community/embeddings/clova.py:50]: could we add docstrings for these

Comment (acho98) [reply libs/community/langchain_community/embeddings/clova.py]: I've added docstrings as requested. The changes have been pushed to the branch.
Thank you for your feedback, @baskaryan

**Review by baskaryan — APPROVED** [review]

**Pull request merged**